### PR TITLE
Fix go report card - gocyclo

### LINKED ISSUE #71: go report card has complexity warnings: https://goreportcard.com/report/github.com/vmware-tanzu/secrets-manager

### PR DESCRIPTION
**Pull request by farhan-pasha**
Fixes the gocyclo problems reported by go report card. Updates docs to properly get pod logs.

Fixes #71

### CODE REVIEW

**Comment by farhan-pasha** [comment]
Go Report Link: https://goreportcard.com/report/github.com/vmware-tanzu/secrets-manager
image

File: docs/_pages/0050-quickstart.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ example-68997489c6-8j8kj   1/1     Running   0          1m51s
 Let’s check the logs of our example workload:
 
 ```bash
-kubectl get logs example-68997489c6-8j8kj
+kubectl logs example-68997489c6-8j8kj
```
Comment (v0lkan) [inline docs/_pages/0050-quickstart.md:169]: Nice catch!

**Review by v0lkan — APPROVED** [review]
LGTM.

**Comment by v0lkan** [comment]
Nice catch about the `kubectl log` command samples.

The rest of the code looks good.

**Pull request merged**